x/gamm/keeper/pool_service_test.go

```diff
@@ -840,29 +842,46 @@ func (suite *KeeperTestSuite) TestJoinSwapExactAmountInConsistency() {
 		shareOutMinAmount sdk.Int
 		expectedSharesOut sdk.Int
 		tokenOutMinAmount sdk.Int
+		expectError       error
 	}{
 		{
-			name:              "single coin with zero swap and exit fees",
+			name:              "happy path: single coin with zero swap and exit fees",
 			poolSwapFee:       sdk.ZeroDec(),
 			poolExitFee:       sdk.ZeroDec(),
```

Comment:
I might be wrong, but as far as I am concerned, the exit fee was removed from osmosis pools via governance recently. I do not know if the governance's decision was already reflected on the protocol, but if yes, I think we should remove the `poolExitFee` field at all here

Reply:
I tried changing it but it doesn't seem to work, I don't think it's necessary to change it yet